WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -483,56 +483,19 @@ import WordPressFlux
 
     // MARK: - Configuration / Topic Presentation
 
-    @objc private func configureStreamHeader() {
```

Comment:
This method shouldn't be removed, as it is required to display the header view for many parts of the Reader.

Reply:
Okay , I see. Nothing was using this method and that's why i deleted it. I will try to fix what you  say.